site/collections/_projects/sustainability-leadership-kosova.md

```diff
@@ -5,8 +5,33 @@ subtitle: Used data to influence an ongoing governmental draft on deforestation
 image: "/images/SKL-project.png"
 ---
 
-<a href="https://www.slkosova.org/">Sustainability Leadership Kosova</a> (SLK) empowers individuals, businesses and organisations in leadership with the aim to cultivate innovative solutions to sustainability challenges. With the help of a local GIS expert, Faruk Foniqi (UN Habitat), SLK has uncovered the rate of deforestation (including illegal deforestation) in Kosovo
+# The Challenge
+[Sustainability Leadership Kosova (SLK)](https://www.slkosova.org/) is a non-profit organization dedicated to empowering individuals, businesses, and organizations to lead and integrate sustainability solutions in Kosovo. Founded in 2019 by Indira Kartallozi and a group of passionate collaborators, SLK aims to drive system change across the economy, people, and nature to create a sustainable, healthy, fair, and prosperous society.
 
-Faruk has used satellite imagery using a GIS model for his analysis and has even made a predictive analysis for future deforestation. The aim of this project was to use this data to influence an ongoing governmental draft on deforestation regulation strategy as well as empower Civil Society Organisations, NGOs, and activists to increase pressure on the government to act.
+One of SLK’s significant challenges has been addressing deforestation in Kosovo. With the assistance of local GIS expert Faruk Foniqi, SLK identified the rate of deforestation, including illegal activities, using satellite imagery and GIS analysis. This data aimed to influence governmental draft regulations on deforestation and empower Civil Society Organisations, NGOs, and activists to pressure the government for action.
 
-There is a now a new blogpost with more details. Find out more about this project [here]({% post_url 2023-06-22-sustainability-leadership-kosova %})
+Despite having the story and the initial data, SLK faced a challenge in effectively presenting this information to raise awareness. This highlights the ongoing need for innovative communication strategies to effectively mobilize support and drive impactful environmental action.
+
+# The Solution
+With the guidance and insights from the team of SLK, the expertise in storytelling of Ashraf Ramzy from MasterStory, and the dedication of CorrelAid volunteers, the combined efforts were focused on transforming the narrative around deforestation in Kosovo. Our volunteers combined data analysis with innovative data visualization techniques to create a compelling and persuasive story. This comprehensive approach turned raw data into meaningful insights, designed to captivate audiences and apply strategic pressure on the government’s deforestation plans. Through this collaboration, SLK, CorrelAidNL, and MasterStory volunteers worked together to breathe life into the data, making it relevant and impactful.
```

Comment:
What about the approach was "comprehensive"? Can you be more specific on who the audience was that was "captivated"? Were the data visualization techniques truly innovative, or were they just data visualization techniques?